src/arcam/fmj/__init__.py

```diff
@@ -235,6 +237,7 @@ class SourceCodes(IntOrTypeEnum):
     USB = 0x0F
     STB = 0x10
     GAME = 0x11
+    PHONO = 0x12
```

Comment:
Added new source as `PHONO` is an available for the SA series ([SA series PDF](https://www.arcam.co.uk/ugc/tor/SA10/Custom%20Installation%20Notes/SH277E_RS232_SA10_SA20_B.pdf) page 9)